src/screens/block/block.vue

```diff
@@ -113,10 +118,22 @@ export default {
       } else if (blockHashRegex.test(this.blockId)) {
         this.$store.dispatch('loadBlock', { hash: this.blockId })
       }
+    },
+    takeAction (event) {
+      if (event.srcKey === 'right' && this.block.height < this.latestHeight) {
+        this.$router.push(`/block/${this.block.height + 1}`)
+      }
+      if (event.srcKey === 'left' && this.block.height > 1) {
+        this.$router.push(`/block/${this.block.height - 1}`)
+      }
+    },
+    getLatestBlockHeight () {
+      this.$store.dispatch('loadLastBlocks')
     }
   },
   mounted () {
     this.getBlock()
+    this.getLatestBlockHeight()
```

Comment:
I think it is more efficient to call `fetchHeight` only if `state.height` is uninitialized.